content/documentation/rpc/wallet/get_notes.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: wallet/getNotes
+description: RPC Wallet | Iron Fish Documentation
+---
+
+Retrieves notes for the given account. If the account is not specified, the default account will be used.
+
+Supports cursor-based pagination using the `pageSize` and `pageCursor` request parameters.
```

Comment:
nit: we could add info about how to get pageCursor: User can use nextPageCursor returned in the response as the page Cursor.

Reply:
Good idea, updated!